e2e/whisper/mailservice_test.go

```diff
@@ -0,0 +1,57 @@
+package whisper
```

Comment:
Why do you need separated test suite for it?

Reply:
Because it's totally independent of other tests. This tests only if `MailService` was properly registered.